django/db/models/query_utils.py

```diff
@@ -27,6 +27,23 @@ class InvalidQuery(Exception):
     pass
 
 
+def merge_dicts(dicts):
+    merged = {}
+    # merge in reverse to preference the order of the original list. e.g.,
+    # merge_dicts([a, b]) will preference the keys in a over those in b.
+    for d in reversed(dicts):
+        merged.update(d)
+    return merged
+
+
+def subclasses(cls):
+    yield cls
+    # python 2 lacks 'yield from'
```

Comment:
Python (capitalize)
Can you expand the comment with what this should be changed to when dropping Python 2 compatibility?